.ci/docker/common/install_cusparselt.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```

Comment:
I think you could add `set -ex` here to harden the script and also allow the bash steps to be printed on GitHub console log to very that the installation works correctly